cmsmon-tools.spec

```diff
@@ -0,0 +1,76 @@
+### RPM cms cmsmon-tools 0.3.10
+## INITENV +PATH PYTHONPATH %i/${PYTHON_LIB_SITE_PACKAGES}
+## NOCOMPILER
+
+%define pkg CMSMonitoring
+%define ver %realversion
+%define cmsmon_commands nats-sub nats-pub nats-exitcodes-termui dbs_vm
+%define flags -ldflags="-s -w -extldflags -static"
```

Comment:
please add `-p %{compiling_processes}` so that we have control over the build processes. Default is to use number of CPUs which we do not want as cmsBuild also build other packages in parallel which might need cpus.

Reply:
done